documentation/memory/step3/using_gc_traces.md

```diff
@@ -1,86 +1,229 @@
 # Tracing garbage collection
 
-There's a lot to learn about how the garbage collector works, but if you learn one thing it's that when GC is running, your code is not.
+This guide will go through the fundamentals of garbage collection traces.
 
-You may want to know how often and how long the garbage collection is running.
+By the end of this guide, you'll be able to:
+* Enable traces in your Node.js application
+* Interpret traces
+* Find memory leak source
```

Comment:
```suggestion
* Interpret traces
* Identify potential memory issues in your Node.js application
```

Rarely, one can find a memory leak by just using `--trace-gc` (in real-world applications)